### PR TITLE
Move onboarding components to project folder

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Move HeadingOld, Card and QuotePriceCard to onboarding folder
- Rename HeadningNew -> Heading

We have a bunch of components in `ui` package that will only ever be used in onboarding, let's move theme to the project folder instead so we don't need to modify all components to use new colors.

**Update** 
Since we used the Card component in `ProductDocuments` I duplicated the Card component and put it back in the theme. That way we don't need to think about breaking styling for onboarding

## Justify why they are needed
`ui` package shouldn't have legacy components

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/onboarding/src/components/Card/SelectableCard.test.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { screen } from '@testing-library/react'
-import { renderWithTheme } from '../../../test-helpers'
+import { renderWithTheme } from 'ui/test-helpers'
```
Comment (robinandeer) [inline apps/onboarding/src/components/Card/SelectableCard.test.tsx:2]: hmm let's try to keep utils to one package, right? But might be okey here

File: apps/onboarding/src/components/Card/SelectableCard.tsx [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { Theme } from '@emotion/react'
 import styled from '@emotion/styled'
 import * as Checkbox from '@radix-ui/react-checkbox'
 import * as RadioGroup from '@radix-ui/react-radio-group'
-import { mq } from '../../lib/media-query'
-import { getColor } from '../../lib/theme/theme'
+import { mq } from 'ui/src/lib/media-query'
+import { getColor } from 'ui/src/lib/theme/theme'
```
Comment (robinandeer) [inline apps/onboarding/src/components/Card/SelectableCard.tsx:6]: Should this not just import from "ui"?

File: apps/onboarding/src/components/QuotePriceCard/QuotePriceCard.tsx [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import styled from '@emotion/styled'
-import { getColor } from '../../lib/theme/theme'
-import { Card, CardContent } from '../Card/Card'
-import { SelectableCard, SelectableCardProps } from '../Card/SelectableCard'
-import { Checkbox } from '../Checkbox/Checkbox'
+import { Checkbox } from 'ui'
+import { Card, CardContent } from 'onboarding/src/components/Card/Card'
```
Comment (robinandeer) [inline apps/onboarding/src/components/QuotePriceCard/QuotePriceCard.tsx:3]: hehe not sure what's going on with the imports 😅

**Review by robinandeer — COMMENTED** [review]
I think most of this is great -- just confused with some of the updated imports

File: apps/onboarding/src/components/QuotePriceCard/QuotePriceCard.tsx (hunk 3 above)
Comment (gustaveen) [reply apps/onboarding/src/components/QuotePriceCard/QuotePriceCard.tsx:3]: ah, it's VSCode that updates the import automatically when moving files :)

File: apps/onboarding/src/components/Card/SelectableCard.test.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/onboarding/src/components/Card/SelectableCard.test.tsx:2]: I hear you! This one might be fine though since it's connected to the theme :)

**Review by guilhermespopolin — APPROVED** [review]

**Comment by gustaveen** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1080?utm_source=stack-comment).


Job Reference: zB8VYvxqwHaQ4wYqK5TQ

**Pull request merged**